runtime/tests/checker/utils.go

```diff
@@ -92,6 +93,12 @@ func ExpectCheckerErrors(t *testing.T, err error, count int) []error {
 
 	require.Error(t, err)
 
+	// Temporary to help catch checking error
+	ee, is := err.(*runtime.ExtendedParsingCheckingError)
+	if is {
+		err = ee.Unwrap()
+	}
+
 	assert.IsType(t, &sema.CheckerError{}, err)
```

Comment:
Same as above, this should use `utils.RequireErrorAs` instead of manually unwrapping

Reply:
`RequireErrorAs` was added in after `0.9` so resolving

Reply:
Ah, right 👍